crates/holochain/src/conductor/conductor/conductor_services/deepkey_service.rs

```diff
@@ -0,0 +1,137 @@
+use std::sync::Arc;
+
+use holo_hash::AgentPubKey;
+use holochain_zome_types::{CellId, Timestamp};
+
+use crate::conductor::ConductorHandle;
+
+/// Interface for the Deepkey service
+#[async_trait::async_trait]
+#[mockall::automock]
+#[allow(clippy::needless_lifetimes)]
+pub trait DeepkeyService: Send + Sync {
```

Comment:
Naming throughout. Deep key is the name of our *specific* dpki implementation. The service traits / reference names should probably be the more general `Dpki`, (e.g. `DpkiService`).

Reply:
This ties in with my question, what is Dpki and why is there sometimes Dpki and sometimes Deepkey?

Reply:
dpki is an acronym for "Distributed Public Key Infrastructure" which is all about making public keys available so you can verify signatures.

If you kinda say DPKI over and over quickly, it sorta sounds like DeepKey, which is where the name for Holo's app implementation of DPKI got its name.